c2corg_ui/static/partials/feed.html

```diff
@@ -1,76 +1,269 @@
+              <div class="bubble" uib-tooltip="{{'height_diff_up'| translate}}">
+                <span class="icon-height_diff"></span>
+              </div>
+            </div>
+            <div class="text-bubble" ng-if="::doc['document']['height_diff_up']">
+              {{::doc['document']['height_diff_up']}} m
+            </div>
+            <div class="parent-bubble" ng-if="::doc['document']['height_diff_difficulties']">
+              <div class="bubble" uib-tooltip="{{'height_diff_difficulties'| translate}}" tooltip-append-to-body="true">
+                <span class="value-title glyphicon glyphicon-resize-vertical"></span>
+              </div>
+            </div>
+            <div class="text-bubble" ng-if="::doc['document']['height_diff_difficulties']">
+              {{::doc['document']['height_diff_difficulties']}} m
+            </div>
+            <div class="grow">
+            </div>
+          </div>
+        </div>
+        <div class="parent-infop parent-infop-first" ng-if="::doc['document']['areas'] != null">
+          <a ng-href="{{::feedCtrl.createURLArea(doc['document']['areas'])}}" title="{{::feedCtrl.showArea(doc['document']['areas'])}}" ng-click="$event.stopPropagation()">
+            <div class="infop-bar">
+              <div class="parent-bubble">
+                <div class="bubble">
+                  <span class="glyphicon glyphicon-map-marker"></span>
+                </div>
+              </div>
+              <div class="text-bubble">
+                {{::feedCtrl.showArea(doc['document']['areas'])}}
+              </div>
+              <div class="grow">
+              </div>
+            </div></a>
+        </div>
+        <div class="parent-infop">
+          <div class="infop-bar">
+
+
+            <div class="parent-bubble" ng-repeat="activity in ::doc['document']['activities']">
+              <a href="#" class="shcom" uib-tooltip="{{activity | translate}}"> 
+                <div class="bubble">
+                  <span class="route-activity icon-{{::activity}}"></span>
+                </div>
+              </a>
+            </div>
+
+            <div class="grow">
+            </div>
+
+
+            <div class="parent-bubble">
+              <a href="#" class="link share-facebook" ng-if="::doc['document']['condition_rating']" uib-tooltip="{{'condition_rating' | translate}} : {{doc['document']['condition_rating'] | translate}}" tooltip-append-to-body="true">
+                <div class="condition-rating" ng-class="::doc['document']['condition_rating']" ></div>
+              </a>
+            </div>
+
+            <div class="parent-bubble"><div class="bubble bquality">
+              <a href="#" class="link share-facebook" uib-tooltip="{{doc['document']['quality'] | translate}}" tooltip-append-to-body="true">
+
+                <span class="quality-{{::doc['document']['quality']}}"></span>
+              </a></div>
```

Comment:
This would be a little more readable to close tags consistently with the opening tags.
```
<div class="parent-bubble"><div class="bubble bquality">
  <a href="#" class="link share-facebook" >
  </a>
</div></div>
```
instead of 
```
<div class="parent-bubble"><div class="bubble bquality">
  <a href="#" class="link share-facebook" >
  </a></div>
</div>
```
Or to have tags on dedicated lines.